web-local/test/ui/dashboards.spec.ts

```diff
@@ -245,6 +245,8 @@ test.describe("dashboard", () => {
     // Filter to Facebook via leaderboard
     await page.getByRole("button", { name: "Facebook 19.3k" }).click();
 
+    await page.waitForTimeout(100);
```

Comment:
Hmmm we should add a smarter wait. This arbitrary wait of 100ms might not actually wait long enough in all cases.

Reply:
Changed to wait for the specific selector